agent/pkg/dns/dns.go

```diff
@@ -1,20 +1,22 @@
 package dns
 
 import (
-	"io/ioutil"
 	"net"
 	"strings"
-	"time"
 
 	mdns "github.com/miekg/dns"
 	"k8s.io/klog/v2"
 
-	beehiveContext "github.com/kubeedge/beehive/pkg/core/context"
 	"github.com/kubeedge/edgemesh/agent/pkg/dns/controller"
 	"github.com/kubeedge/edgemesh/common/util"
 )
 
-const hostResolv = "/etc/resolv.conf"
+func (dns *EdgeDNS) Run() {
```

Comment:
Just keep `Run` as the original code position?

Reply:
I just want to be consistent with the location of the Run function in other files, such as: tunnel.go, proxy.go